docs/synthetics/ci-cd/github-actions.md

```diff
@@ -1,13 +1,30 @@
 title: GitHub Actions Integration
 description: Guide on how to integrate Sematext Synthetics with GitHub Actions.
 
+## Configure repository variables for the integration
+The GitHub Actions integration works by running a Job which runs a Synthetics monitor of your choice against the desired URL. In order to specify which monitor this will be, as well as the API key required to run it, you need to configure some repository variables first.
```

Comment:
The GitHub Actions integration works by running a Job which runs a Synthetics monitor
==> The GitHub Actions integration works by running a Synthetics monitor

Reply:
Yeah, I didn't like the original myself but it was technically more accurate than the short version which is why I kept it. Alright, I'll change it 👍